ix-dev/community/audiobookshelf/templates/test_values/basic-values.yaml

```diff
@@ -9,9 +9,13 @@ audiobookshelf:
   additional_envs: []
 network:
   host_network: false
+  enable_opds: true
```

Comment:
lets remove this, and depend on the bind_mode of opds_port.
ie `if values.network.opds_port.bind_mode` ...